### PR TITLE
finished refactor to re-render after post 

### PR DESCRIPTION
**Pull request by itsnormsyo**
finished refactor to re-render after post also fixed comments from previous commit

### CODE REVIEW

File: client/components/Tracker/index.jsx [hunk 1]
```diff
@@ -19,9 +26,13 @@ const Tracker = () => {
 
         Banner
       </Banner>
-      <Form />
+      <Form getList={() => getList()} />
```
Comment (jessicaawhy) [inline client/components/Tracker/index.jsx:29]: `getList={getList}` can be used instead here, I believe.

File: client/components/Tracker/index.jsx [hunk 2]
```diff
@@ -19,9 +26,13 @@ const Tracker = () => {
...
 
-      <StyledList />
+      <StyledList
+        thoughtsList={thoughtsList}
+        getList={() => getList()}
```
Comment (jessicaawhy) [inline client/components/Tracker/index.jsx:33]: Same comment as before

File: client/components/Tracker/index.jsx [hunk 3]
```diff
@@ -19,9 +26,13 @@ const Tracker = () => {
...
+        setThoughtsList={() => setThoughtsList()}
```
Comment (jessicaawhy) [inline client/components/Tracker/index.jsx:34]: Same comment. `getList={getList}` & `setThoughtsList={setThoughtsList}`

**Review by jessicaawhy — DISMISSED** [review]
Great work - renders perfectly. Another consideration for later - should we be rendering in reverse order?

File: client/components/Tracker/index.jsx (hunk 1 above)
Comment (itsnormsyo) [reply client/components/Tracker/index.jsx:29]: resolved Joseph will push changes. Thanks Jessica!

File: client/components/Tracker/index.jsx (hunk 3 above)
Comment (itsnormsyo) [reply client/components/Tracker/index.jsx:34]: resolved Joseph will push changes. Thanks Jessica!

**Review by jessicaawhy — APPROVED** [review]
WIP Tracker functionality rerenders after post request. Approved.

**Pull request merged**